commands/completion-scripts/completion.bash.gotmpl

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+
+: {{.App.Name}}
```

Comment:
`.App.Name` is the name of the app. For `dnscontrol`, this is hard-coded in [commands/commands.go:53](https://github.com/StackExchange/dnscontrol/blob/e32bdc053f64efc78a9f1fc3ac25cd8af6737425/commands/commands.go#L53), but if that line is removed, it is the name of the running binary.

It's useful not to hardcode the app name in the completion script in case the binary is named differently - that way, completions are generated for the specific name they run the tool with.

This usage of `:` essentially just functions as a comment (literally a noop) [[gnu.org](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Bourne-Shell-Builtins)]. A lot of bash completion scripts I've come across have that at the top.